zap/zap.gradle.kts

```diff
@@ -154,7 +154,8 @@ val japicmp by tasks.registering(JapicmpTask::class) {
     fieldExcludes = listOf()
 
     classExcludes = listOf(
-        "org.zaproxy.zap.extension.custompages.ContextCustomPagePanel\$CustomPagesMultipleOptionsPanel")
+        "org.zaproxy.zap.extension.custompages.ContextCustomPagePanel\$CustomPagesMultipleOptionsPanel",
+        "org.parosproxy.paros.core.scanner.Variant#setParameters(HttpMessage, List)")
```

Comment:
I'd expect this to be in the method excludes.

Reply:
I am not able to exclude this method with following string: `org.parosproxy.paros.core.scanner.Variant#setParameters(org.parosproxy.paros.network.HttpMessage,java.util.List)`

can you please help with this?

Reply:
got it.please ignore.

Reply:
There's a classpath issue that prevents the check from working when added to the method excludes, add just the class. Not worth keeping this waiting for that.

Reply:
I fixed it for now.

Reply:
It's preferable not to add that in this PR, it might break other plugins.

Reply:
oh ... ok